src/codegen/utilities/interpreter_helpers.py

```diff
@@ -46,6 +46,8 @@
     "get_analog_power_up_states_with_output_type",
 ]
 
+CTYPE_CHANGE_FUNCTIONS = ["read_digital_lines", "write_digital_lines"]
```

Comment:
This is a very generic name for a very specific set of functions. In this case, I think you might as well check against the specific function names in the if-statement or use something like `if "digital_lines" in function.name`.

Reply:
Added the logic Scarpigen https://github.com/ni/grpc-device-scrapigen/pull/172.